src/modules/75-cd/components/PipelineSteps/KubernetesInfraSpec/KubernetesInfraSpec.tsx

```diff
@@ -112,7 +112,7 @@ const KubernetesInfraSpecEditable: React.FC<KubernetesInfraSpecEditableProps> =
         validationSchema={validationSchema}
         onSubmit={noop}
       >
-        {formik => {
+        {(formik: any) => {
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/harness/harness-core-ui/pullRequest?prid=9232293)
```suggestion
        {(formik: unknown) => {
```